doc/topics/eauth/index.rst

```diff
@@ -285,6 +293,15 @@ the results are filtered against ``auth.ldap.groupclass``, default
 
     auth.ldap.groupou: Groups
 
+Note that as of 2017.7, auth.ldap.groupclass can refer to either a groupclass or an objectClass.
+For some LDAP servers (notably OpenLDAP without the `memberOf` overlay enabled) to determine group
+membership we to know both the `objectClass` and the `memberUid` attributes.  Usually for these
```

Comment:
`want` to know? `need` to know?

Also, if you want objectClass (and other references like that) to show up in the docs as code references rather than plain text, they need two backticks. It's a funny markDown vs restructuredText quirk. :)